app/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesJob.scala

```diff
@@ -49,8 +51,10 @@ class PollingNewMessagesJob @Inject() (
       .flatMap { lastActivityMap =>
         lastActivityMap.toSeq.traverse { case (ern, lastActivity) =>
           if (shouldUpdateMessages(lastActivity, fastIntervalTime, fastCutoffTime, slowIntervalTime)) {
+            logger.warn(s"[PollingNewMessagesJob] - Updating messages for ern: $ern")
             messageService.updateMessages(ern)
           } else {
+            logger.warn(s"[PollingNewMessagesJob] - No messages to update for ern: $ern")
```

Comment:
Sorry Nemanja, I don't think we need more logging here.  The work that @wolfendale has been doing with the jobs includes logging

Reply:
Do you mean in whole file or just on this line :) ?

Reply:
Done :)